src/Tools/ExternalAccess/OmniSharp/Completion/OmniSharpCompletionService.cs

```diff
@@ -23,7 +23,7 @@ internal static class OmniSharpCompletionService
             CancellationToken cancellationToken = default)
             => completionService.GetCompletionsInternalAsync(document, caretPosition, trigger, roles, options, cancellationToken);
 
-        public static string GetProviderName(this CompletionItem completionItem) => completionItem.ProviderName;
+        public static string? GetProviderName(this CompletionItem completionItem) => completionItem.ProviderName;
```

Comment:
This change looks good to me. Luckily, while it definitely surprises me, I wasn't depending on provider name being non-null in O#'s service.